src/AutoComplete/StyledAutoCompleteClearIndicator.js

```diff
@@ -3,23 +3,17 @@ import PropTypes from 'prop-types';
 
 import Icon from '../Icon';
 
-const StyledAutoCompleteClearIndicator = ({
-  innerProps,
-  innerRef,
-  ...props
-}) => (
+const StyledAutoCompleteClearIndicator = ({ innerProps, ...props }) => (
   <Icon
     aria-label="clear selection"
     fontSize="size4"
     name="cross"
-    ref={innerRef}
     {...{ ...innerProps, ...props }}
   />
 );
 
 StyledAutoCompleteClearIndicator.propTypes = {
-  innerProps: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
-  innerRef: PropTypes.func.isRequired
```

Comment:
react-select [doesn't pass innerRef to its ClearIndicator](https://github.com/JedWatson/react-select/blob/3c7de0de52826fe74d303a01475c43fe88256156/packages/react-select/src/Select.js#L1556), so this always just spews warnings